test/lint/streams.rb

```diff
@@ -90,6 +90,14 @@ def test_xadd_with_maxlen_and_approximate_option
       assert_match ENTRY_ID_FORMAT, actual
     end
 
+    def test_xadd_with_nomkstream_option
+      actual = redis.xadd('s1', { f1: 'v1', f2: 'v2' }, nomkstream: true)
+      assert_equal nil, actual
```

Comment:
```suggestion
      assert_nil actual
```

Otherwise minitest will emit a warning.

Reply:
Updated in aae269592bc5b30eb2a9927c5e58ac225e375efc.